src/components/va-alert/va-alert.css

```diff
@@ -23,9 +23,17 @@ div.alert {
   padding: 3.2rem 6.4rem 3.2rem 2.4rem;
   width: 100%;
   vertical-align: middle;
+  box-sizing: border-box;
+}
+
+div.alert:not(.fullwidth):not(.bg-only) {
   border-left-style: solid;
   border-left-width: 10px;
-  box-sizing: border-box;
+}
+div.alert.fullwidth.info,
+div.alert.fullwidth.warning {
+  border-top-style: solid;
+  border-top-width: 10px;
```

Comment:
The `.info` and `.warning` are low effort ways of enforcing this bullet point from [the docs](https://design.va.gov/components/alertboxes#more-about-full-width-alerts):

> Only available in `info` or `warning` variants.

I don't think we need to spend a lot of of time right now coming up with some strategy involving throwing errors or ensuring that certain props are mutually exclusive.

Reply:
So we're defining a misuse of the component here as using `fullwidth` with an alert type of anything other than `info` or `warning`, right? If the component is misused, what should happen? What does happen?

Reply:
As far as what should happen goes, I don't know :shrug:. We could log an error message and hope that a dev sees it, but they might not. They would be even less likely to see it if the props are dynamic in any way. We could also not render anything if the props are incompatible with each other. However, what _does_ happen is we get no border at all. Here is an example of what happens with `status="continue"`:

![image](https://user-images.githubusercontent.com/2008881/134557009-da8c64d8-561b-4482-9333-1ccb303d67d7.png)